templates/around/map.html

```diff
@@ -5,8 +5,8 @@
 {% block extra_css %}
   <link rel=stylesheet href={{ url_for('static', filename='css/around-map.css') }}>
   <!-- leaflet -->
-  <link rel=stylesheet href=https://unpkg.com/leaflet@1.2.0/dist/leaflet.css integrity=sha512-M2wvCLH6DSRazYeZRIm1JnYyh22purTM+FDB5CsyxtQJYeKq83arPe5wgbNmcFXGqiSH2XR8dT/fJISVA1r/zQ==/>
-  <script src="https://unpkg.com/leaflet@1.2.0/dist/leaflet.js" integrity="sha512-lInM/apFSqyy1o6s89K4iQUKg6ppXEgsVxT35HbzUupEVRh2Eu9Wdl4tHj7dZO0s1uvplcYGmt3498TtHq+log=="></script>
+  <link rel=stylesheet href=https://unpkg.com/leaflet@1.2.0/dist/leaflet.css integrity=sha512-M2wvCLH6DSRazYeZRIm1JnYyh22purTM+FDB5CsyxtQJYeKq83arPe5wgbNmcFXGqiSH2XR8dT/fJISVA1r/zQ== crossorigin=/>
```

Comment:
`crossorigin=`?
I'm not sure what this does, however it should have quotes `"` for it's empty value, or have no `=` at all!

Reply:
without `=` we have an error

Reply:
In which case you could then say `crossorigin=""`.
What's the error message BTW?